app/src/features/charts/TableChart.tsx

```diff
@@ -23,52 +25,43 @@ const StyledTableHeader = withStyles((theme: Theme) =>
   })
 )(TableCell);
 
-function TableChart(props: { data: Row[]; fields?: Field[] }) {
-  let tableColumns: Field[] | undefined = undefined;
-  if (props.data.length > 0) {
-    tableColumns =
-      props.fields === undefined
-        ? Object.keys(props.data[0]).map((name) => ({
-            name: name,
-            displayName: name,
-          }))
-        : props.fields;
+function TableChart(props: { data: Row[]; fields?: string[] }) {
+  const tableColumns: string[] | undefined =
+    !props.fields && props.data.length > 0
+      ? Object.keys(props.data[0])
+      : props.fields;
+
+  function getDisplayName(field: string) {
+    if (Object.keys(BREAKDOWN_VAR_DISPLAY_NAMES).includes(field)) {
+      return BREAKDOWN_VAR_DISPLAY_NAMES[field as BreakdownVar];
+    } else if (Object.keys(VARIABLE_DISPLAY_NAMES).includes(field)) {
+      return VARIABLE_DISPLAY_NAMES[field as VariableId];
+    }
+    return field;
```

Comment:
Can probably put this into the DisplayNames file?

Reply:
oh ya good call!